### DISCUSSION TITLE
Override a view

### DISCUSSION THREAD

**Discussion by ejaz-alam-dev** [bagisto/bagisto#7599] [Q&A]
Hello Everyone,

I want to override a view file. I followed this tutorial `https://bagisto.com/en/how-to-override-view-files-in-bagisto/`.
in provider class, I added my view file location as 

`$this->publishes([
            __DIR__ . '/../Resources/views/shop/layouts/top-nav/login-section.blade.php' => resource_path('themes/velocity/views/shop/layouts/top-nav/login-section.blade.php')
        ]);`
        
        
but its not working. I am confused that what should I put in the `resource_path` because I am overriding webkul package view file.

**Comment by Amitk-webkul** [comment]
@ejaz-alam-dev, In this code, you are specifying that the original file is located at` __DIR__. '/../Resources/views/shop/layouts/top-nav/login-section.blade.php'` should be published to resource_path('`themes/velocity/views/shop/layouts/top-nav/login-section.blade.php`').

You've run the `php artisan vendor:publish` command after making this change to publish the view file to the correct location.
